### ISSUE TITLE
SSH interactive terminal

### ISSUE THREAD

**Issue by DiegoVillegasA** [berkeley-dsep-infra/hubs#2845]
Hi,

I am trying to access datahub.berkeley.edu through my terminal in order to expand the memory limit of jupyterhub from 1GB to 2.5GB to import a large dataset. As the UC Berkeley JupyterHubs Doc suggested, after obtaining an authentication token, I try to run:

ssh <hub-username>@<hub-location>

in my case:
ssh diego_villegas_a@datahub.berkeley.edu

and I receive the following error: 
ssh: connect to host datahub.berkeley.edu port 22: Connection timed out

I have tried it outside and on campus, I have checked in my app features that SSH Client is activated, and I do not know what else to do. 

Any help would be highly appreciated!

**Comment by balajialg** [comment]
@DiegoVillegasA Thanks for raising the request! Not sure whether I am answering your question directly but can you fill this request [form](https://github.com/berkeley-dsep-infra/datahub/issues/new?assignees=balajialg&labels=support&template=higher-resources.md&title=Request+more+RAM+for+class+X) for increasing the RAM? As a policy, every RAM increase request would go through approval at our end. You can refer more to this documentation [here](https://ds-modules.github.io/curriculum-guide/faq/changerequirements.html).


In the meantime, someone from the team will respond to your SSH-related query!

**Comment by DiegoVillegasA** [comment]
Thank you very much!

**Comment by yuvipanda** [comment]
Thank you for opening this, @DiegoVillegasA. Unfortunately we removed the experimental ssh support recently, as we didn't have enough time to dedicate to developing that service. With https://github.com/berkeley-dsep-infra/datahub/pull/2849, I've removed references to it from our docs too. Sorry for the confusion.

However, even with that service, it gave you the same setup using the hub via the browser did - there was no way to increase resources there (other than asking here!)
